cdc/processor/manager.go

```diff
@@ -159,11 +159,11 @@ func (m *managerImpl) Tick(stdCtx context.Context, state orchestrator.ReactorSta
 	return state, nil
 }
 
-func (m *managerImpl) closeProcessor(changefeedID model.ChangeFeedID, ctx cdcContext.Context) {
+func (m *managerImpl) closeProcessor(changefeedID model.ChangeFeedID) {
```

Comment:
Would you please add some comment to explain that this function may stuck the tick function?

Reply:
I add comment for `processor.Tick` and `processor.Close`, which can be called in `processor.Manager.Tick`.